src/Containers/Microsoft.NET.Build.Containers/ContainerBuilder.cs

```diff
@@ -2,11 +2,22 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using Microsoft.NET.Build.Containers.Resources;
+using System.Text;
 
 namespace Microsoft.NET.Build.Containers;
 
 public static class ContainerBuilder
 {
+
+    private static void WriteLogMessage(LogMessage message) {
```

Comment:
very minor: private methods should follow public members.

Reply:
the method was removed in favor of `ILogger` implementation